__tests__/utils/fonts.test.js

```diff
@@ -19,7 +19,7 @@ describe('Util font should correct', () => {
     })
 
    it('normalizeFontSize should run correctly', () => {
-        expect(normalizeFontSize(14)).toEqual(5.447470817120623)
+        expect(normalizeFontSize(14)).toEqual(12.600000000000001)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case for `normalizeFontSize` function has been updated to expect a different value. This change is fine as long as the logic of `normalizeFontSize` function has been changed accordingly and the new expected value is correct. However, it's worth noting that comparing floating point numbers directly can lead to precision issues. It would be better to use a function like `toBeCloseTo` for this comparison.
```diff
-        expect(normalizeFontSize(14)).toEqual(12.600000000000001)
+        expect(normalizeFontSize(14)).toBeCloseTo(12.6)
```
This will ensure that the test doesn't fail due to minor differences in precision.


<!-- This is an auto-generated comment by OSS CodeRabbit -->